pkg/ext-proc/handlers/request.go

```diff
@@ -0,0 +1,140 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+
+	configPb "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	filterPb "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/ext_proc/v3"
+	extProcPb "github.com/envoyproxy/go-control-plane/envoy/service/ext_proc/v3"
+	klog "k8s.io/klog/v2"
+
+	"ext-proc/scheduling"
+)
+
+const (
+	targetPodHeader = "target-pod"
```

Comment:
should we make this a config value passed in at startup, with `target-pod` as the default? since this value is controlled externally

Reply:
sounds like a good idea, made it an optional argument